src/options/didyoumean_test.cpp

```diff
@@ -426,340 +426,339 @@ set<string> getDebugTags()
+      "strings-lb",
+      "strings-fmf",
+      "no-strings-fmf",
+      "strings-eit",
+      "no-strings-eit",
+      "strings-alphabet-card",
+      "show-sat-solvers",
+      "random-freq",
+      "random-seed",
+      "restart-int-base",
+      "restart-int-inc",
+      "refine-conflicts",
+      "no-refine-conflicts",
+      "minisat-elimination",
+      "no-minisat-elimination",
+      "minisat-dump-dimacs",
+      "no-minisat-dump-dimacs",
+      "model-format",
+      "dump",
+      "dump-to",
+      "force-logic",
+      "simplification",
+      "no-simplification",
+      "static-learning",
+      "no-static-learning",
+      "produce-models",
+      "no-produce-models",
+      "check-models",
+      "no-check-models",
+      "dump-models",
+      "no-dump-models",
+      "proof",
+      "no-proof",
+      "check-proofs",
+      "no-check-proofs",
+      "dump-proofs",
+      "no-dump-proofs",
+      "produce-unsat-cores",
+      "no-produce-unsat-cores",
+      "produce-assignments",
+      "no-produce-assignments",
+      "interactive",
+      "no-interactive",
+      "ite-simp",
+      "no-ite-simp",
+      "on-repeat-ite-simp",
+      "no-on-repeat-ite-simp",
+      "simp-with-care",
+      "no-simp-with-care",
+      "simp-ite-compress",
+      "no-simp-ite-compress",
+      "unconstrained-simp",
+      "no-unconstrained-simp",
+      "repeat-simp",
+      "no-repeat-simp",
+      "simp-ite-hunt-zombies",
+      "sort-inference",
+      "no-sort-inference",
+      "incremental",
+      "no-incremental",
+      "abstract-values",
+      "no-abstract-values",
+      "model-u-dt-enum",
+      "no-model-u-dt-enum",
+      "tlimit",
+      "tlimit-per",
+      "rlimit",
+      "rlimit-per",
+      "rewrite-apply-to-const",
+      "no-rewrite-apply-to-const",
+      "replay",
+      "replay-log",
+      "decision",
+      "decision-threshold",
+      "decision-use-weight",
+      "no-decision-use-weight",
+      "decision-random-weight",
+      "decision-weight-internal",
+      "version",
+      "license",
+      "help",
+      "show-config",
+      "show-debug-tags",
+      "show-trace-tags",
+      "early-exit",
+      "no-early-exit",
+      "threads",
+      "threadN",
+      "filter-lemma-length",
+      "fallback-sequential",
+      "no-fallback-sequential",
+      "incremental-parallel",
+      "no-incremental-parallel",
+      "no-interactive-prompt",
+      "continued-execution",
+      "immediate-exit",
+      "segv-spin",
+      "no-segv-spin",
+      "segv-nospin",
+      "wait-to-join",
+      "no-wait-to-join",
+      "strict-parsing",
+      "no-strict-parsing",
+      "mmap",
+      "no-mmap",
+      "no-checking",
+      "no-filesystem-access",
+      "no-include-file",
+      "enable-idl-rewrite-equalities",
+      "disable-idl-rewrite-equalities",
+      "sets-propagate",
+      "no-sets-propagate",
+      "sets-eager-lemmas",
+      "no-sets-eager-lemmas",
+      NULL,
+  }; /* cmdlineOptions */
   int i = 0;
```

Comment:
I actually thinks it's better :-) Maybe we should discuss what we could use for introducing consistent formatting (automated) at one point.